AUTHORS

```diff
@@ -1,7 +1,2 @@
-Maintainers
------------
-Full Name <email>
-
-Contributors
-------------
-Full Name <email>
+Andrea Manzi <andrea.manzi@gmail.com>
```

Comment:
Here it would have been better to stick to the format of the AUTORS from our template. Or we update the template if we agree it's not fine.

Reply:
ups, going to fix it